jobs/build/promote/Jenkinsfile

```diff
@@ -327,25 +333,42 @@ node {
                     }
 
                     prevMinor = minor - 1
-                    commonlib.inputRequired(taskThread) {
-                        def resp = input(
-                            message: "${acquire_failure}What PREVIOUS releases should be included in ${release_name} (arch: ${arch})?",
-                            parameters: [
-                                string(
-                                    defaultValue: "4.${prevMinor}.?",
-                                    description: "This is release ${release_name}. What release is in flight for the previous minor release 4.${prevMinor}?",
-                                    name: 'IN_FLIGHT_PREV',
-                                ),
-                                string(
-                                    defaultValue: "${suggest_previous}",
-                                    description: (acquire_failure?acquire_failure:"Doozer thinks these are the other releases to include.") + " Edit as necessary (comma delimited).",
-                                    name: 'SUGGESTED',
-                                ),
-                            ]
-                        )
+                    in_flight_prev_required = true
+                    if (params.IN_FLIGHT_PREV) {
+                        in_flight_prev = params.IN_FLIGHT_PREV.trim()
+                        pattern = /4\.$prevMinor\.(\d+)/
+                        match = in_flight_prev =~ pattern
+                        match.find()
+                        if (match.size() == 1) {
```

Comment:
Would also like a way to specify an empty `in_flight_prev`. e.g. currently we always leave that input box empty if no 4.5.z release in flight.

Reply:
added an option of "skip"